gr-uhd/lib/usrp_source_impl.cc

```diff
@@ -57,8 +57,10 @@ usrp_source_impl::usrp_source_impl(const ::uhd::device_addr_t& device_addr,
 
     _samp_rate = this->get_samp_rate();
     _samps_per_packet = 1;
-    register_msg_cmd_handler(cmd_tag_key(),
-                             [this](const pmt::pmt_t& val, int chan, const pmt::pmt_t& msg) { this->_cmd_handler_tag(val); });
+    register_msg_cmd_handler(
+        cmd_tag_key(), [this](const pmt::pmt_t& val, int chan, const pmt::pmt_t& msg) {
+            this->_cmd_handler_tag(val);
+        });
```

Comment:
I assume with this change you can remove the `functional` include at the top of the file.